app/graphql/schema.graphql

```diff
@@ -650,7 +650,7 @@ type Rule implements Node {
   description: String
   failedCount: Int
   id: ID!
-  identifier: String
+  identifier: JSON
```

Comment:
Check with frontend.

Would this return JS Objects or a string of JSON?

Reply:
https://github.com/RedHatInsights/compliance-frontend/pull/1836